src/stacks-hierarchy/Root_ChooseTicketReceiverScreen/Root_ChooseTicketReceiverScreen.tsx

```diff
@@ -18,11 +19,19 @@ export const Root_ChooseTicketReceiverScreen: React.FC<Props> = ({
   route: {params},
 }) => {
   const styles = useStyles();
-
   const {t} = useTranslation();
-
   const focusRef = useFocusOnLoad();
 
+  const {mutateAsync: getAccountIdByPhone} = useGetAccountIdByPhoneMutation();
+  const handleValidatePhoneNumber = async (phoneNumber: string) => {
+    try {
+      const result = await getAccountIdByPhone(phoneNumber);
+      return result;
+    } catch (error) {
+      return error as string;
+    }
+  };
+
```

Comment:
I think this can be simplified to
```suggestion
const handleValidatePhoneNumber = async (phoneNumber: string) =>
    await getAccountIdByPhone(phoneNumber);
```

Reply:
Or maybe you can even use `getAccountIdByPhone` directly, omitting `handleValidatePhoneNumber` altogether?

Reply:
Yeah apparently that works! Thanks for the suggestion!